modules/indexing/src/main/scala/ch/epfl/bluebrain/nexus/kg/indexing/organizations/OrganizationEsIndexer.scala

```diff
@@ -50,13 +51,17 @@ class OrganizationEsIndexer[F[_]](client: ElasticClient[F],
   private val deprecatedKey        = "deprecated".qualifyAsString
   private val orgName              = "name".qualifyAsString
   private lazy val indexJson: Json = jsonContentOf("/es-index.json", Map(quote("{{type}}") -> t))
+  private val indices              = Set[String]()
```

Comment:
This data structure is not thread safe, which could result in arbitrary failures in concurrent cases. Are concurrent cases possible of all indexer types (org / domain / schema / context / instance)?

Reply:
I assume we don't, but could you confirm?